### ISSUE TITLE
Duplicate quick-fix when using default literal: Upgrade this project to C# language version 'default'

### ISSUE THREAD

**Issue by phrohdoh** [dotnet/roslyn#29640]
I do not know if this is the correct repo or if this should be reported elsewhere (csharplang, omnisharp, etc).

**Version Used**: 

VSCode: 1.26.1 `493869ee8e8a846b0855873886fc79d480d342de`

Omnisharp extension: 1.15.2

```shell
$ dotnet --info

.NET Core SDK (reflecting any global.json):
 Version:   2.1.401
 Commit:    91b1c13032

Runtime Environment:
 OS Name:     Mac OS X
 OS Version:  10.13
 OS Platform: Darwin
 RID:         osx.10.13-x64
 Base Path:   /usr/local/share/dotnet/sdk/2.1.401/

Host (useful for support):
  Version: 2.1.3
  Commit:  124038c13e

.NET Core SDKs installed:
  2.1.401 [/usr/local/share/dotnet/sdk]

.NET Core runtimes installed:
  Microsoft.AspNetCore.All 2.1.3 [/usr/local/share/dotnet/shared/Microsoft.AspNetCore.All]
  Microsoft.AspNetCore.App 2.1.3 [/usr/local/share/dotnet/shared/Microsoft.AspNetCore.App]
  Microsoft.NETCore.App 2.1.3 [/usr/local/share/dotnet/shared/Microsoft.NETCore.App]

To install additional .NET Core runtimes or SDKs:
  https://aka.ms/dotnet-download
```

**Steps to Reproduce**:

I have an uninitialized generic local

```csharp
        private async Task<TApiResp> MakeApiCall<TApiResp>(string apiPath, AuthFlow auth)
        {
            _client.SetupHeaderAuth(auth.Token);

            TApiResp response = default;
            // ... more code here ...
            return response;
       }
```

When hovering over the `default` literal I get this tooltip:

```
Feature 'default literal' is not available in C# 7.0. Please use language version 7.1 or greater.
```

*Two* of the `Upgrade this project to C# language version 'default'` quick-fixes are available.

**Expected Behavior**:

Only a single quick-fix of this type is suggested.

**Actual Behavior**:

Multiple are suggested.

<img width="601" alt="screen shot 2018-09-01 at 5 14 00 pm" src="https://user-images.githubusercontent.com/4861023/44951844-227e4380-ae35-11e8-8746-8386ff7b716d.png">

**More information**:

Relevant JSON from the Omnisharp log:

```json
{
  "Request_seq": 15524,
  "Command": "/v2/getcodeactions",
  "Running": true,
  "Success": true,
  "Message": null,
  "Body": {
    "CodeActions": [
      {
        "Identifier": "Upgrade this project to C# language version 'default'",
        "Name": "Upgrade this project to C# language version 'default'"
      },
      {
        "Identifier": "Upgrade this project to C# language version 'default'",
        "Name": "Upgrade this project to C# language version 'default'"
      },
      {
        "Identifier": "Introduce local for 'default'",
        "Name": "Introduce local for 'default'"
      },
      {
        "Identifier": "Introduce local for all occurrences of 'default'",
        "Name": "Introduce local for all occurrences of 'default'"
      }
    ]
  },
  "Seq": 10750,
  "Type": "response"
}
```

**Comment by CyrusNajmabadi** [comment]
I don't repro this either.  closing out.
